Content.Server/NukeOps/System/WarConditionOnExamineSystem.cs

```diff
@@ -0,0 +1,65 @@
+using Robust.Shared.Timing;
+
+namespace Content.Server.NukeOps
+{
+    /// <summary>
+    /// Shows information about war conditions on examine
+    /// </summary>
+    [UsedImplicitly]
+    public sealed class WarConditionOnExamineSystem : EntitySystem
+    {
+        [Dependency] private readonly NukeopsRuleSystem _nukeopsRuleSystem = default!;
+        [Dependency] private readonly IGameTiming _gameTiming = default!;
+        [Dependency] private readonly PowerReceiverSystem _powerReceiverSystem = default!;
+
+        public override void Initialize()
+        {
+            base.Initialize();
+            SubscribeLocalEvent<WarConditionOnExamineComponent, ExaminedEvent>(OnExamine);
+        }
+
+        private void OnExamine(EntityUid uid, WarConditionOnExamineComponent component, ExaminedEvent args)
+        {
+            // This component applied to comms console so we must check is it have power to work
+            if (!_nukeopsRuleSystem.RuleAdded || !_powerReceiverSystem.IsPowered(uid))
+            {
+                return;
+            }
+
+            var condition = _nukeopsRuleSystem.GetWarCondition();
+            switch (condition)
+            {
+                case WarConditionStatus.NO_WAR_UNKNOWN:
+                    args.PushMarkup($"{Loc.GetString("war-ops-examine-status-unable")}\n{Loc.GetString("war-ops-examine-conditions-unknown")}");
+                    break;
+                case WarConditionStatus.NO_WAR_SHUTTLE_DEPARTED:
+                    args.PushMarkup($"{Loc.GetString("war-ops-examine-status-unable")}\n{Loc.GetString("war-ops-examine-conditions-left-outpost")}");
+                    break;
+                case WarConditionStatus.NO_WAR_TIMEOUT:
+                    args.PushMarkup($"{Loc.GetString("war-ops-examine-status-unable")}\n{Loc.GetString("war-ops-examine-conditions-time-out")}");
+                    break;
+                case WarConditionStatus.NO_WAR_SMALL_CREW:
+                    args.PushMarkup($"{Loc.GetString("war-ops-examine-status-unable")}\n{Loc.GetString("war-ops-examine-conditions-small-crew", ("min_size", _nukeopsRuleSystem.Config.WarMinCrewSize))}");
+                    break;
+                case WarConditionStatus.YES_WAR:
+                    var gameruleTime = _gameTiming.CurTime.Subtract(_nukeopsRuleSystem.GameruleStartTime);
+                    var timeLeft = _nukeopsRuleSystem.Config.WarDeclarationTimeWindow.Subtract(gameruleTime);
+                    args.PushMarkup($"{Loc.GetString("war-ops-examine-status-able")}\n{Loc.GetString("war-ops-examine-conditions-timer", ("minutes", timeLeft.Minutes), ("seconds", timeLeft.Seconds))}");
+                    break;
+                case WarConditionStatus.WAR_DELAY:
+                    var timeAfterDeclaration = _gameTiming.CurTime.Subtract(_nukeopsRuleSystem.DeclarationTime);
+                    var timeRemain = _nukeopsRuleSystem.Config.WarNukieArriveDelay.Subtract(timeAfterDeclaration);
```

Comment:
Naively checking times breaks with server snapshots because it doesn't consider serialized times.

Reply:
what?

Reply:
A lot of this code won't work with replays none of it considers pausing.